### ISSUE TITLE
Insufficient contrast ratios between background and foreground colours

### ISSUE THREAD

**Issue by matthall8** [FACN4/BSN_week1-group-project#19]
The website failed the accessibility issue on the grounds of the background and foreground not having sufficient contrast ratios.

![image](https://user-images.githubusercontent.com/9437789/42309986-e3f64eb6-8042-11e8-8a97-d063975e5998.png)
